src/utils/gethRead.js

```diff
@@ -10,6 +11,12 @@ const nodesmithSupported = !!NODESMITH_KEY
 
 const callbacks = {}
 
+const nodesmithOpenPromise = new Promise(resolve => {
+  nodesmithProvider.onopen = () => {
+    resolve()
+  }
+})
+
```

Comment:
1 liner
`const nodesmithOpenPromise = new Promise(resolve => nodesmithProvider.onopen = resolve)`

Reply:
eslint complains that you shouldn't return assignment